nf_core/__main__.py

```diff
@@ -1622,6 +2014,56 @@ def bump_version(new_version, dir, nextflow):
         sys.exit(1)
 
 
+# nf-core create-logo
+
+
+@nf_core_cli.command("create-logo")
+@click.argument("logo-text", metavar="<logo text>")
+@click.option("-d", "--dir", type=click.Path(), default=".", metavar="<directory>")
+@click.option(
+    "--theme",
+    type=click.Choice(["light", "dark"]),
+    default="light",
+    help="Theme for the logo.",
+    show_default=True,
+)
+@click.option(
+    "--width",
+    type=int,
+    default=2300,
+    help="Width of the logo in pixels. Will be overwritten if `logo-text` is longer than the given width.",
```

Comment:
> Will be overwritten if `logo-text` is longer than the given width.

Should the entire image not be resized to fit into this width if the text is long? We're resizing anyway, right?

Reply:
yeah, sorry this is outdated.